.env

```diff
@@ -34,6 +34,10 @@ MAIL_PORT="25"
 SMTP_USERNAME="ofn"
 SMTP_PASSWORD="f00d"
 
+OPENID_APP_ID="4461746120466f6f6420436f6e736f727469756d"
+OPENID_APP_SECRET="secret"
+OPENID_REDIRECT_URI="http://localhost:3000/user/spree_user/auth/openid_connect/callback"
```

Comment:
This needs to move to ofn-install, right?

Reply:
Yes, ofd-install is a better place to move it to before merging, although `OPENID_APP_ID` and `OPENID_APP_SECRET` will be the same across all instances eventually. The `OPENID_REDIRECT_URI` probably doesn't need to be a variable at all, it's just the host + that path `/user/spree_user/auth/openid_connect/callback` and also shouldn't change across instances either.

Reply:
I've removed `OPENID_REDIRECT_URI`; the other two variables are already set if defined in the secrets file via https://github.com/openfoodfoundation/ofn-install/pull/766. FR-staging has the secrets defined. 

For consistency, should we keep the two dummy values in the `.env` file since we do the same for other variables? 

> Maybe we can introduce a new structure in ofn-secrets with a common file from which all other configs inherit

Are there other examples where the values are the same across instances? If this is the first time we've encountered this, it might be more work than it's worth, but if we're going to encounter this a lot then we could potentially set up a shared file. With them being encrypted though I'm guessing it won't be straightforward.

Reply:
Great. Keeping the current env vars is fine. The shared vars was just an idea. You are right to not implement that yet. And I like to keep the examples in the .env file, too.